ci-operator/step-registry/ibmcloud/provision/bastionhost/ibmcloud-provision-bastionhost-commands.sh

```diff
@@ -191,3 +196,7 @@ echo "${proxy_private_url}" > "${SHARED_DIR}/proxy_private_url"
 # echo proxy IP to ${SHARED_DIR}/proxyip
 echo "${bastion_public_ip}" > "${SHARED_DIR}/proxyip"
 
+#OCP-60816 - [IPI-on-IBMCloud] install cluster under BYON with a different resource group	
+if [ -s "${SHARED_DIR}/ibmcloud_cluster_resource_group" ]; then
+    getResources > "${SHARED_DIR}/vpc_resources"
```

Comment:
I do not think  we should use "${SHARED_DIR}/ibmcloud_cluster_resource_group" to decide if collect vpc resource list, `ibmcloud_cluster_resource_group` might also used in other case, not only for OCP-60816. Or maybe you can develop a separate ref step to run the command, then insert it into your wanted install workflowl

Reply:
1. if not use "${SHARED_DIR}/ibmcloud_cluster_resource_group" for this case, how about use CREATE_CLUSTER_RESOURCE_GROUP? then I will add CREATE_CLUSTER_RESOURCE_GROUP to ibmcloud-provision-bastionhost-commands.sh step and other steps.
2. add a separate step? not use  CREATE_CLUSTER_RESOURCE_GROUP env?